src/test/java/net/barrage/school/java/ecatalog/app/RequestSenderTest.java

```diff
@@ -0,0 +1,33 @@
+package net.barrage.school.java.ecatalog.app;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.autoconfigure.actuate.observability.AutoConfigureObservability;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.test.context.ActiveProfiles;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+@ActiveProfiles("fake")
+@AutoConfigureObservability
+public class RequestSenderTest {
+
+    @Value("${loader.endpoint}")
+    private String endpoint;
+
+    @Value("${loader.requestsPerMinute}")
+    private int requestsPerMinute;
+
+    @Test
+    public void sendRequests() throws InterruptedException {
+        TestRestTemplate restTemplate = new TestRestTemplate();
+
+
+        for (int i = 0; i < requestsPerMinute; i++) {
```

Comment:
Didn't get it a bit... If I want to create 100 RPM pressure, how much time I have to wait?)

Reply:
I sleep between every request for 
`Thread.sleep(60000 / requestsPerMinute);`

Reply:
Sorry my bad) Yes, its fine!